front/pages/edit.vue

```diff
@@ -30,27 +30,78 @@
         <div
           class="
             flex flex-col
+            mx-auto
             m-2 md:mx-4 md:mb-4 md:ml-0
             p-4
             bg-white
             rounded-lg
             h-100
             justify-between
           "
+          style="max-width: 500px"
         >
-            <div class="flex flex-1 m-4 justify-center">
-              <AppButton
-                size="medium"
-                v-on:click="Overlay()"
-                class="text-white my-auto"
-                :style="myPixelButtonStyle"
-              >
-                Votre pixel: {{ x + 1 }}:{{ y + 1 }}
-              </AppButton>
+
+          <div class="flex-col justify-between flex-1 flex sm:hidden">
```

Comment:
Maybe use `md:` instead of `sm:` to match layout's breakpoint

(also at line 58)